string movement_type          # "pose", "joint", "named", "cartesian"
geometry_msgs/Pose pose_target
string named_target
float64[] joint_values
float64[] start_joint_values
manymove_planner/MovementConfig config